### PR TITLE
Updated code block style in 3.11 Scaling and Performance guide

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -28,17 +28,29 @@ xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#no
 == Listing nodes
 To list all nodes that are known to the master:
 
+[source,terminal]
 ----
 $ oc get nodes
+----
+
+.Example output
```
Comment (bobfuru) [inline admin_guide/manage_nodes.adoc:36]: I believe someone noticed that we used TitleCase in 3.11 😾  - calling it out here if you want to update throughout.

File: scaling_performance/host_practices.adoc [hunk 2]
```diff
@@ -41,23 +42,24 @@ kubernetesMasterConfig:
...
+Prometheus and adjust the `maxRequestsInFlight` and `QPS` accordingly.
+
+There needs to be a good balance when changing the default values as the CPU and memory consumption of
+API server and etcd IOPS will increase when it is handling more requests in parallel. Also note that heavy
+non-watch requests might overload the API server as they get cancelled after a fixed 60 second timeout and
+the client starts retrying.
+
+Provided sufficient CPU and memory resources are available on the API server system(s), the API server
+requests overloading issue can be safely alleviated by taking into account the factors mentioned above and
+bumping up the maxRequestsInFlight, API qps and burst values in the *_/etc/origin/master/master-config.yamlyaml
```
Comment (bobfuru) [inline scaling_performance/host_practices.adoc:60]: Looks like you have `yaml` twice in the file extension

**Review by bobfuru — COMMENTED** [review]
Couple of minor things, otherwise LGTM!

File: admin_guide/manage_nodes.adoc (hunk 1 above)
Comment (ahardin-rh) [reply admin_guide/manage_nodes.adoc:36]: Ohhh! Good catch. Updating throughout.

**Pull request merged**